Curve-finance-suite/Unkill/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "curve-12",
+  "version": "0.0.1",
+  "description": "This Agent detects when Curve Pool Owner calls unkill_me function",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "bignumber.js": "^9.0.1",
+    "forta-agent": "^0.0.32",
+    "forta-agent-tools": "^1.0.31"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
```

Comment:
`@types/nodemon` also not used.

Reply:
Actually, it is used in `start:dev` command